packages/cid-verifier/test/verification.spec.js

```diff
@@ -1,17 +1,24 @@
+// @ts-ignore
+import Hash from 'ipfs-only-hash'
 import { test, getMiniflare } from './utils/setup.js'
 import { toDenyListAnchor } from '../src/utils/denylist.js'
 
+/**
+ * @param {string} s
+ */
+const createTestCid = async (s) => (new Buffer(await Hash.of(s))).toString('base64')
+
 // TODO: use valid cids and test 400 scenarios
-const cidInDenyList = 'asdfasdf'
-const cidInDenyListBlockedForLeganReasons = 'blocked for legal reasons'
-const pendingCid = 'pending'
-const emptyCid = 'empty'
-const notMalwareCid = 'notMalware'
-const malwareCid = 'malware'
-const newCid = 'newCid'
-const maliciousUrl = encodeURIComponent('http://malicious/url')
-const safeUrl = encodeURIComponent('http://safe/url')
-const errorUrl = encodeURIComponent('http://error/url')
+const cidInDenyList = await createTestCid('asdfasdf')
```

Comment:
@vasco-santos i'd like to understand why this isn't working.  can you help me?  i'm getting this from `normalizeCid` when it gets these CIDs: `Unexpected end of data`.  will be on in about 6 hours to address and wrap this up.

Reply:
CID is not a random string, it has a format that it must follow. The error `Unexpected end of data` is definitely not great 😓 .

There are a lot of details around what a CID is, and for that I recommend you to read its spec https://github.com/multiformats/cid

In super short from spec above:

The format of a CID is `<cidv1> ::= <multibase-prefix><multicodec-cidv1><multicodec-content-type><multihash-content-address>
`

A CID is a Self-describing content-addressed identifiers for distributed systems. This means that it can be encoded in any way and have any type of hashing algorithm. So, it must be constructed in a way that it can be decoded and has within it all the information for properly parse it.

So, there are two options here:
- create CIDs randomly using something like ipfs binary `ipfs add --only-hash --cid-version 1 file.txt` and hardcode them - I usually just do this
- rely on a module like https://github.com/alanshaw/ipfs-only-hash to just create CIDs from random strings in the code

Reply:
thanks!  `{ cidVersion: 1}` worked!